### PR TITLE
Tutorial explorer: New layout for specific locale tutorial set

### PR DESCRIPTION
**Pull request by breville**
It was observed that the set of images in the tutorial explorer had a lot of en text, and might have given non-en visitors the impression that the tutorials are not localised.  However, a lot of them are, and we in fact have an "Activities in your language" UI that shows locale-specific tutorials for non-en visitors.  (The full set is available below, after pressing a "Show activities for many languages" button to expand it, and has not been changed.)

To address this, the layout of the "Activities in your language" tutorial set has been changed, with the images being reduced in size and the localised text increased in size.

Going forward, we will attempt to acquire more images without text; if we get enough, we might be able to revert to the previous layout with much bigger images again.

## full width

Screenshot 2019-09-27 09 46 22

## medium width

Screenshot 2019-09-27 09 47 18

Truncating the title with ellipses has been disabled.  This does mean that a long title could push the text outside of the regular bounding box, but the layout seems to survive just fine, as seen here with an earlier font size:

### Left:

Screenshot 2019-09-25 18 21 19

### Right:

Screenshot 2019-09-25 18 21 58

### CODE REVIEW

File: apps/src/tutorialExplorer/tutorialSet.jsx [hunk 1]
```diff
@@ -62,14 +64,23 @@ export default class TutorialSet extends React.Component {
...
-            filters={this.props.filters}
-            key={item.code}
-            tutorialClicked={this.tutorialClicked.bind(this, item)}
-          />
-        ))}
+        {this.props.tutorials.map(item =>
+          this.props.specificLocale ? (
+            <TutorialSpecificLocale
+              item={item}
+              filters={this.props.filters}
```
Comment (islemaster) [inline apps/src/tutorialExplorer/tutorialSet.jsx]: `TutorialSpecificLocale` doesn't take the `filters` prop.

Comment (islemaster) [reply apps/src/tutorialExplorer/tutorialSet.jsx]: Or, if you wanted these to be interoperable, maybe we should use them as such...

```jsx
const TutorialComponent = this.props.specificLocale ? TutorialSpecificLocale : Tutorial;
//...
{this.props.tutorials.map(item => (
  <TutorialComponent
    // ...
  />
))}
```
...and DRY the props-passing.

Comment (breville) [reply apps/src/tutorialExplorer/tutorialSet.jsx]: Great idea, will do!

**Review by islemaster — APPROVED** [review]
:+1: Simplified even further.  LGTM!

**Comment by agealy** [comment]
Thanks for taking the time to tweak the font sizing, @breville!

**Pull request merged**